mlx/ops.cpp

```diff
@@ -1316,6 +1316,16 @@ array mean(
     const std::vector<int>& axes,
     bool keepdims /* = false */,
     StreamOrDevice s /* = {}*/) {
+  int ndim = a.ndim();
+  for (int axis : axes) {
+    if (axis < -ndim || axis >= ndim) {
+      std::ostringstream msg;
+      msg << "[mean] axis " + std::to_string(axis) +
+              " is out of bounds for array with " + std::to_string(ndim) +
+              " dimensions.";
```

Comment:
Hate to be a stickler about this again, but could you change the operation from `+` to `<<` 

```
      msg << "[mean] axis " << axis << " is out of bounds for array with " << ndim << " dimensions.";
```

Reply:
gotcha!